vagrant/boxes.d/00-centos.yaml

```diff
@@ -19,3 +19,8 @@ boxes:
     box_name: centos7-atomic
     pty: true
     libvirt: http://cloud.centos.org/centos/7/atomic/images/CentOS-Atomic-Host-7-Vagrant-Libvirt.box
+
+  google-centos7:
+    box_name: 'google/gce'
+    google_options:
+      image_family: 'centos-7'
```

Comment:
I am not seeing why this box needs its own definition and name? Google is just a provider, so you should be able to specify this on the main `centos7` box.

Reply:
The main centos7 box only supports 4 providers? https://app.vagrantup.com/centos/boxes/7

Reply:
Some provides do not use box names. openstack also uses an image name. You already override the box in the library so that can be ignored.

Reply:
I'm pretty sure the google provider requires you to set box_name to 'google/gce' for any box and use image_name to set the os.

Reply:
Oh, I guess in the past, we've just used overrides? Is that the right way to go?

Reply:
Isn't that why you have `override.vm.box = 'google/gce'` in `configure_google_provider`?